dags/covid_etl_dag.py

```diff
@@ -22,6 +23,10 @@
     conf.get('core', 'dags_folder') + "/../notebooks") + "/"
 OUTPUT_FOLDER = os.path.abspath(
     conf.get('core', 'dags_folder') + "/../output") + "/"
+QA_FOLDER = os.path.abspath(
+    conf.get('core', 'dags_folder') + "/../snowflake/qa") + "/"
+GITUSER = Variable.get('GIT_USER')
```

Comment:
Could we add here a default value in case the user is not specified? Later on, we can check if this variable is properly set and skip GitHub ticket opening when it's not